src/views/product-downloads-view/components/featured-tutorials-section/featured-tutorials-section.module.css

```diff
@@ -1,49 +1,7 @@
 .sectionHeading {
   color: var(--token-color-foreground-strong);
-  margin-bottom: 12px;
 }
 
-/*
-TODO: these @media queries come from the numbers defined in src/pages/style.css
-but I wasn't able to get the @media --dev-dot-mobile and @media --dev-dot-tablet
-queries to work like they do in that file. The code below will eventually need
-to be refactored, this is a temporary copy/paste to get the new install view
-out.
-*/
 .cardGrid {
```

Comment:
Now using `CardsGridList`, which handles the styling needed here.

Reply:
This was probably satisfying to remove 😎

Reply:
Haha, twas indeed 😬🔪